<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mqttclient</class>
 <widget class="QWidget" name="mqttclient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1159</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mqttclient</string>
  </property>
  <property name="styleSheet">
   <string notr="true">widget.setStyleSheet(&quot;border: 1px solid black;&quot;)</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>连接配置</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>数据发送</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>数据接收</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>301</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget
{
	border:1px solid black;
}
QLabel,QCheckBox,QTextBrowser
{
	border:0px solid black;
}
QPushButton
{
    /*背景颜色 
    background-color:transparent;*/
    /*字体颜色为白色    
    color:black; */
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,stop:0 #497BF0,stop:1 #1FB6F6);
}
QPushButton:pressed
{
    /*按下时字向右移动4像素*/  
    padding-left:4px;
    /*按下时字向下移动4像素*/  
    padding-top:4px;
	
	background-color: rgb(85, 255, 0);
}</string>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>IP:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>端口号:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ClientId:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>用户名:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>密码:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>连接状态:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="CleanSession">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Clean  Session</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>心跳时间:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_IP">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Port">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ClientId">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_username">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>120</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_password">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>150</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_time">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>260</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_conn">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>180</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>连接</string>
    </property>
   </widget>
   <widget class="QLabel" name="Status_label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>180</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>btn_conn</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>label_7</zorder>
   <zorder>label_8</zorder>
   <zorder>label_9</zorder>
   <zorder>CleanSession</zorder>
   <zorder>label_10</zorder>
   <zorder>lineEdit_IP</zorder>
   <zorder>lineEdit_Port</zorder>
   <zorder>lineEdit_ClientId</zorder>
   <zorder>lineEdit_username</zorder>
   <zorder>lineEdit_password</zorder>
   <zorder>lineEdit_time</zorder>
   <zorder>Status_label</zorder>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>30</y>
     <width>411</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget
{
	border:1px solid black;
}
QLabel
{
	border:0px solid black;
}
QPushButton
{
    /*背景颜色 
    background-color:transparent;*/
    /*字体颜色为白色    
    color:black; */
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,stop:0 #497BF0,stop:1 #1FB6F6);
}
QPushButton:pressed
{
    /*按下时字向右移动4像素*/  
    padding-left:4px;
    /*按下时字向下移动4像素*/  
    padding-top:4px;
	
	background-color: rgb(85, 255, 0);
}</string>
   </property>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>发布的主题:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_pub">
    <property name="geometry">
     <rect>
      <x>319</x>
      <y>70</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>发送数据</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_pub_clear">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>120</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>清除</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_pub">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>210</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>清除</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_publish">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>19</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_publish">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>301</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_pub">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>301</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>731</width>
     <height>271</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget
{
	border:1px solid black;
}
QLabel
{
	border:0px solid black;
}
QPushButton
{
    /*背景颜色 
    background-color:transparent;*/
    /*字体颜色为白色    
    color:black; */
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,stop:0 #497BF0,stop:1 #1FB6F6);
}
QPushButton:pressed
{
    /*按下时字向右移动4像素*/  
    padding-left:4px;
    /*按下时字向下移动4像素*/  
    padding-top:4px;
	
	background-color: rgb(85, 255, 0);
}</string>
   </property>
   <widget class="QPushButton" name="btn_sub">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>20</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>订阅</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_sub_clear">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>20</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>清除</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_subscribe">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>521</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_recv">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>701</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_4" native="true">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>30</y>
     <width>141</width>
     <height>591</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget
{
	border:1px solid black;
}</string>
   </property>
   <widget class="QPushButton" name="btn_openled">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
                   background-color:rgba(100,225,100,30);
                   border-style:outset;                
                   border-width:4px;                     
                   border-radius:10px;                   
                   border-color:rgba(255,255,255,30);   
                   font:bold 10px;                      
                   color:rgba(0,0,0,100);               
                   padding:6px;                          
				
}
QPushButton:pressed{
                   background-color:rgba(100,255,100,200);
                   border-color:rgba(255,255,255,30);
                   border-style:inset;
                   color:rgba(0,0,0,100);
				 /*按下时字向右移动2像素*/  
    				padding-left:2px;
    				/*按下时字向下移动2像素*/  
   	 			padding-top:2px;
}
QPushButton:hover{
                   background-color:rgba(100,255,100,100);
                   border-color:rgba(255,255,255,200);
                   color:rgba(0,0,0,200);
}</string>
    </property>
    <property name="text">
     <string>开灯</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_closeled">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
                   background-color:rgba(100,225,100,30);
                   border-style:outset;                
                   border-width:4px;                     
                   border-radius:10px;                   
                   border-color:rgba(255,255,255,30);   
                   font:bold 10px;                      
                   color:rgba(0,0,0,100);               
                   padding:6px;                          
				
}
QPushButton:pressed{
                   background-color:rgba(100,255,100,200);
                   border-color:rgba(255,255,255,30);
                   border-style:inset;
                   color:rgba(0,0,0,100);
				 /*按下时字向右移动2像素*/  
    				padding-left:2px;
    				/*按下时字向下移动2像素*/  
   	 			padding-top:2px;
}
QPushButton:hover{
                   background-color:rgba(100,255,100,100);
                   border-color:rgba(255,255,255,200);
                   color:rgba(0,0,0,200);
}</string>
    </property>
    <property name="text">
     <string>关灯</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_opencurtain">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
                   background-color:rgba(100,225,100,30);
                   border-style:outset;                
                   border-width:4px;                     
                   border-radius:10px;                   
                   border-color:rgba(255,255,255,30);   
                   font:bold 10px;                      
                   color:rgba(0,0,0,100);               
                   padding:6px;                          
				
}
QPushButton:pressed{
                   background-color:rgba(100,255,100,200);
                   border-color:rgba(255,255,255,30);
                   border-style:inset;
                   color:rgba(0,0,0,100);
				 /*按下时字向右移动2像素*/  
    				padding-left:2px;
    				/*按下时字向下移动2像素*/  
   	 			padding-top:2px;
}
QPushButton:hover{
                   background-color:rgba(100,255,100,100);
                   border-color:rgba(255,255,255,200);
                   color:rgba(0,0,0,200);
}</string>
    </property>
    <property name="text">
     <string>开窗帘</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_closecurtain">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
                   background-color:rgba(100,225,100,30);
                   border-style:outset;                
                   border-width:4px;                     
                   border-radius:10px;                   
                   border-color:rgba(255,255,255,30);   
                   font:bold 10px;                      
                   color:rgba(0,0,0,100);               
                   padding:6px;                          
				
}
QPushButton:pressed{
                   background-color:rgba(100,255,100,200);
                   border-color:rgba(255,255,255,30);
                   border-style:inset;
                   color:rgba(0,0,0,100);
				 /*按下时字向右移动2像素*/  
    				padding-left:2px;
    				/*按下时字向下移动2像素*/  
   	 			padding-top:2px;
}
QPushButton:hover{
                   background-color:rgba(100,255,100,100);
                   border-color:rgba(255,255,255,200);
                   color:rgba(0,0,0,200);
}</string>
    </property>
    <property name="text">
     <string>关窗帘</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_increase">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
                   background-color:rgba(100,225,100,30);
                   border-style:outset;                
                   border-width:4px;                     
                   border-radius:10px;                   
                   border-color:rgba(255,255,255,30);   
                   font:bold 10px;                      
                   color:rgba(0,0,0,100);               
                   padding:6px;                          
				
}
QPushButton:pressed{
                   background-color:rgba(100,255,100,200);
                   border-color:rgba(255,255,255,30);
                   border-style:inset;
                   color:rgba(0,0,0,100);
				 /*按下时字向右移动2像素*/  
    				padding-left:2px;
    				/*按下时字向下移动2像素*/  
   	 			padding-top:2px;
}
QPushButton:hover{
                   background-color:rgba(100,255,100,100);
                   border-color:rgba(255,255,255,200);
                   color:rgba(0,0,0,200);
}</string>
    </property>
    <property name="text">
     <string>提高光照阈值</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_decrease">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>380</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
                   background-color:rgba(100,225,100,30);
                   border-style:outset;                
                   border-width:4px;                     
                   border-radius:10px;                   
                   border-color:rgba(255,255,255,30);   
                   font:bold 10px;                      
                   color:rgba(0,0,0,100);               
                   padding:6px;                          
				
}
QPushButton:pressed{
                   background-color:rgba(100,255,100,200);
                   border-color:rgba(255,255,255,30);
                   border-style:inset;
                   color:rgba(0,0,0,100);
				 /*按下时字向右移动2像素*/  
    				padding-left:2px;
    				/*按下时字向下移动2像素*/  
   	 			padding-top:2px;
}
QPushButton:hover{
                   background-color:rgba(100,255,100,100);
                   border-color:rgba(255,255,255,200);
                   color:rgba(0,0,0,200);
}</string>
    </property>
    <property name="text">
     <string>降低光照阈值</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_history">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>450</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
                   background-color:rgba(100,225,100,30);
                   border-style:outset;                
                   border-width:4px;                     
                   border-radius:10px;                   
                   border-color:rgba(255,255,255,30);   
                   font:bold 10px;                      
                   color:rgba(0,0,0,100);               
                   padding:6px;                          
				
}
QPushButton:pressed{
                   background-color:rgba(100,255,100,200);
                   border-color:rgba(255,255,255,30);
                   border-style:inset;
                   color:rgba(0,0,0,100);
				 /*按下时字向右移动2像素*/  
    				padding-left:2px;
    				/*按下时字向下移动2像素*/  
   	 			padding-top:2px;
}
QPushButton:hover{
                   background-color:rgba(100,255,100,100);
                   border-color:rgba(255,255,255,200);
                   color:rgba(0,0,0,200);
}</string>
    </property>
    <property name="text">
     <string>历史数据</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>20</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>系统控制</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_his" native="true">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>160</y>
     <width>211</width>
     <height>361</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget
{
	border:1px solid black;
}</string>
   </property>
   <widget class="QPushButton" name="btn_select">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
                   background-color:rgba(100,225,100,30);
                   border-style:outset;                
                   border-width:4px;                     
                   border-radius:10px;                   
                   border-color:rgba(255,255,255,30);   
                   font:bold 10px;                      
                   color:rgba(0,0,0,100);               
                   padding:6px;                          
				
}
QPushButton:pressed{
                   background-color:rgba(100,255,100,200);
                   border-color:rgba(255,255,255,30);
                   border-style:inset;
                   color:rgba(0,0,0,100);
				 /*按下时字向右移动2像素*/  
    				padding-left:2px;
    				/*按下时字向下移动2像素*/  
   	 			padding-top:2px;
}
QPushButton:hover{
                   background-color:rgba(100,255,100,100);
                   border-color:rgba(255,255,255,200);
                   color:rgba(0,0,0,200);
}</string>
    </property>
    <property name="text">
     <string>查询</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_his">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>191</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="dateTimeEdit_start">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="dateTimeEdit_end">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <zorder>widget_4</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>widget</zorder>
  <zorder>widget_3</zorder>
  <zorder>widget_2</zorder>
  <zorder>label_11</zorder>
  <zorder>widget_his</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
